node/14/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM debian:buster-slim
```

Comment:
Should we use debian 11 (bullseye) ? https://wiki.debian.org/LTS support for buster will end in 2024 so its ok to keep it

Reply:
We will use Node 14 image for PIM v6, which still uses Debian Buster: https://github.com/akeneo/pim-enterprise-dev/blob/v6.0.26/Dockerfile#L5

Let's keep Buster for Node 14, and we may use Bullseye for next versions